cmd/clusterawsadm/api/bootstrap/v1alpha1/types.go

```diff
@@ -96,6 +96,8 @@ type EKSConfig struct {
 	// AllowIAMRoleCreation controls whether the EKS controllers have permissions for creating IAM
 	// roles per cluster
 	AllowIAMRoleCreation bool `json:"iamRoleCreation,omitempty"`
+	// EnableEKSViewNodesAndWorkloadsPolicy controls the creation of the policy to view EKS nodes and workloads.
+	EnableEKSViewNodesAndWorkloadsPolicy bool `json:"enableEKSViewNodesAndWorkloadsPolicy"`
```

Comment:
Too long?
```suggestion
	EnableNodesAndWorkloadsPolicy bool `json:"enableEKSViewNodesAndWorkloadsPolicy"`
```

Also is this omitempty?

Reply:
Updated the name and added omitempty